primitives/src/validation_function_params.rs

```diff
@@ -0,0 +1,51 @@
+//! Validation Function Parameters govern the ability of parachains to upgrade their validation functions.
```

Comment:
License header missing.

Reply:
I think I need a VS Code extension that nags me if the license header is missing; I just don't think of that at first.